### DISCUSSION TITLE
Party idea

### DISCUSSION THREAD

**Discussion by liexes** [diamond-rip/Eden#349] [Ideas]
Include the possibility that the plugin is compatible with the alessiodp parties plugin, it could be made easy just making the party data be saved in the database, so the parties plugin is connected and eden to the same database (MongoDb) and so the party data is shared and it is easier without the need to edit the code, it only stores the party data in the database, obviously you must place the same prefixes or locations of the parties plugin to That is how they are shared, I hope and take this idea into account since it is something that will greatly improve the plugin, this is an idea that no arenapvp or practicepvp plugin has.

Plugin: https://alessiodp.com/parties

**Comment by Chromaah** [comment]
I also wanted PAF support but GoodestEnglish said that it probably won't be added, and if he maybe decides to, probably not anytime soon

**Comment by ghost** [comment]
@GoodestEnglish The owner of the Parties plugin says that he would like to work with you to make it possible for the plugin to be compatible with yours (This is my other account Liexes ia the first)

**Reply by GoodestEnglish** [reply]
It mainly not about the compatible. Adding support to party is alright. It is mainly about the time. There's some suggested feature which is more useful and is already waited for 3 months already. Here's the list of all todos https://github.com/diamond-rip/Eden/discussions/categories/to-do-list

**Reply by ghost** [reply]
It's understandable, I'm just asking for this feature to be added, it's something I'm serious, it's something that can make the plugin grow too much, I just ask, what this function is added, what is compatible with this plugin, take advantage, the owner of that plugin wants to work with you, it doesn't matter if it takes 3 months,  I just want it to be added. Do you understand me?
